### PR TITLE
Add geotope_kantone_stand

### PR DESCRIPTION
**Pull request by marionb**
Test link: https://mf-chsdi3.dev.bgdi.ch/ltmbr_geotope_kantone_stand/shorten/8c00755510

### CODE REVIEW

**Comment by ltclm** [comment]
@marionb is this pr ready for review or still WIP?

**Comment by marionb** [comment]
Sorry I forgot to change the title. @ltclm it is ready for a review.

**Review by ltclm — APPROVED** [review]
thank you @marionb 
great work!

**Comment by marionb** [comment]
This layer has been postponed to the December deploy.

**Comment by marionb** [comment]
here the new test link: https://mf-chsdi3.dev.bgdi.ch/ltmbr_geotope_kantone_stand/shorten/8d25ffdd6e

File: chsdi/templates/htmlpopup/geotope_kantone_stand.mako [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<tr><td class="cell-left">${_(layer + 'name')}</td> <td>${c['attributes']['name'] or '-'}</td></tr>
+<tr><td class="cell-left">${_(layer + 'sigel')}</td> <td>${c['attributes']['sigel'] or '-'}</td></tr>
+<tr><td class="cell-left">${_(layer + 'inventar')}</td> <td>${c['attributes']['inventar'] or '-'}</td></tr>
+<tr><td class="cell-left">${_(layer + 'zugang')}</td> <td>${c['attributes']['zugang'] or '-'}</td></tr>
+
+<%
+  url_path = c['attributes']['link'] or '-'
+%>
+<tr>
+<td class="cell-left">${_(layer + 'link')}</td>
```
Comment (b4l) [inline chsdi/templates/htmlpopup/geotope_kantone_stand.mako:16]: There exists a generic link translation that you can use here.

**Review by b4l — CHANGES_REQUESTED** [review]

**Review by b4l — APPROVED** [review]

**Review by ltclm — APPROVED** [review]

**Pull request merged**